core/Cargo.toml

```diff
@@ -56,9 +56,9 @@ serde_json = "1.0"
 prost-build = "0.11"
 
 [features]
-secp256k1 = [ "libsecp256k1" ]
-ecdsa = [ "p256" ]
-rsa = [ "dep:ring" ]
+secp256k1 = ["libsecp256k1"]
+ecdsa = ["p256"]
+rsa = ["dep:ring"]
```

Comment:
No unrelated reformatting please :)

Reply:
I wasn't ware I did this. Maybe `cargo fmt`?

Reply:
Unlikely, `cargo fmt` doesn't yet do `Cargo.toml` files!

Reply:
Oh, then I have no idea.